pkg/types/types.go

```diff
@@ -277,6 +283,16 @@ type CA struct {
 	Password   string `yaml:"password,omitempty"`
 }
 
+type Thanos struct {
+	Disabled              bool     `yaml:"disabled,omitempty"`
+	Version               string   `yaml:"version,omitempty"`
```

Comment:
For required fields, remote the omitempty
For other fields add comments to each field to descript its purpose and possible options so that they show up in the docs.